cli/src/main/resources/logback.xml

```diff
@@ -8,8 +8,8 @@
     </encoder>
   </appender>
 
-  <root level="INFO">
+  <root>
```

Comment:
Intellij complaint about this and said this attribute was not defined. However, this change caused massive log-spam.
```suggestion
  <!-- Setting the default threshold to INFO to avoid logspam. Please ignore IntelliJ lies that the attribute level is undefined. -->
  <root level="INFO">
```